smdebug/tensorflow/keras.py

```diff
@@ -711,6 +714,134 @@ def _remove_fetches_and_callbacks(self, mode):
             x.fetch_callbacks.pop(tf_obj)
         self._fetches_added.clear()
 
+    def _decrement_step(self):
+        """
+        Called when both profiler and debugger are enabled in the native training loop
+        to adjust the step number
+        """
+
+        self.step -= 1
+        self.mode_steps[self.mode] -= 1
+
+        # Decrease Global step number irrespective of what mode it is
+        if self.mode != ModeKeys.GLOBAL:
+            self.mode_steps[ModeKeys.GLOBAL] = self.step
+
+    def _handle_start_python_profiling(self, mode):
+        """
+        This function is called to handle python profiling at the start of a step.
+        :param mode: ModeKeys.TRAIN ModeKeys.EVAL ModeKeys.PREDICT
+        :return:
```

Comment:
Let's remove all of these `:return:` from the docstrings since nothing is being returned.